Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -1152,9 +1152,11 @@
 
 #if HAVE(SANDBOX_STATE_FLAGS)
 (with-filter (require-not (state-flag "WebContentProcessLaunched"))
-    (allow syscall-unix (syscall-unix-only-in-use-during-launch)))
+    (allow syscall-unix (syscall-unix-only-in-use-during-launch))
+    (allow sysctl-read (sysctl-name "vm.malloc_ranges"))) ;; <rdar://problem/105161083>
```

Comment:
Thanks! In order to sort the sandbox by operation, I think it would be good to put this close to the other sysctl-reads.

Reply:
Will change.